contracts/multisig/package.json

```diff
@@ -0,0 +1,93 @@
+{
+    "private": true,
+    "name": "@0x/contracts-multisig",
+    "version": "1.0.0",
+    "engines": {
+        "node": ">=6.12"
+    },
+    "description": "Smart contract components of 0x protocol",
+    "main": "lib/src/index.js",
+    "directories": {
+        "test": "test"
+    },
+    "scripts": {
+        "build": "yarn pre_build && tsc -b",
+        "build:ci": "yarn build",
+        "pre_build": "run-s compile generate_contract_wrappers",
```

Comment:
Since we are now committing the `artifacts` and `abi-gen-wrappers` to Git, we don't _need_ to compile and generate the wrappers every time we build. The assumption is that the package ships with these (and actually re-generating them in CI, will hide the fact that they aren't up-to-date, no?). Let's discuss this issue, it's not clear cut.

Reply:
Addressed offline